src/__test__/Mission.test.js

```diff
@@ -0,0 +1,21 @@
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import '@testing-library/jest-dom';
+import { Provider } from 'react-redux';
+import Missions from '../components/Missions';
+import store from '../redux/configureStore';
+
+describe('Missions tests', () => {
+  test('should render', () => {
+    const missions = render(
+      <Provider store={store}>
+        <BrowserRouter>
+          <Missions />
+        </BrowserRouter>
+        ,
+      </Provider>,
+    );
+
+    expect(missions).toMatchSnapshot();
+  });
+});
```

Comment:
- [ ] one of the tests is failing please make sure that tests are passing in order to complete the requirements 
<img width="890" alt="Screenshot 2023-01-05 at 14 26 20" src="https://user-images.githubusercontent.com/61790587/210790904-87fda7d5-3116-48fd-86e1-e24a6725d427.png">

Reply:
Thanks for reviewing. Added new snapshot.